eslzArm/eslz-portal.json

```diff
@@ -3845,6 +3845,2334 @@
                         }
                     ]
                 },
+                {
+                    "name": "workloadspecific",
+                    "label": "Workload Specific Compliance",
```

Comment:
Sorry, we may of discussed this before but it may well of slipped my mind on the outcome.

Do we think that this should be next to and after the regulatory compliance blade, as it feels a little disconnceted and overlapping in the current blade layout/

Reply:
It's not, it's between "Landing Zones..." and "Decommissioned/Sandbox".